tests/func/test_add.py

```diff
@@ -57,6 +57,21 @@ def test_add(tmp_dir, dvc):
     }
 
 
+def test_add_executable(tmp_dir, dvc):
+    tmp_dir.dvc_gen_exec({"foo": "foo"})
```

Comment:
This is the only place where `dvc_gen_exec` is used, better to just inline it here

Reply:
I inlined `dvc_gen_exec` code here.